{"name": "test_b", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "self = <other.demo_allure.TestAllure object at 0x00000259B69C0130>\n\n    @allure.story(\"测试b\")\n    @allure.severity(allure.severity_level.CRITICAL)\n    def test_b(self):\n        print('test_')\n>       assert False\nE       assert False\n\ndemo_allure.py:34: AssertionError"}, "start": 1646746471228, "stop": 1646746471228, "uuid": "94337149-69b9-42f2-ac3a-e8eae4ceff3b", "historyId": "1ecfcb43f4ef3bde5ccc3d944b1f2302", "testCaseId": "1d7901090f334c5a1eaead3146631f21", "fullName": "demo_allure.TestAllure#test_b", "labels": [{"name": "story", "value": "测试b"}, {"name": "severity", "value": "critical"}, {"name": "feature", "value": "allure测试"}, {"name": "suite", "value": "demo_allure"}, {"name": "subSuite", "value": "TestAllure"}, {"name": "host", "value": "zhh-matebook14"}, {"name": "thread", "value": "1900-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "demo_allure"}]}